components/Collections/src/MDCCollectionViewFlowLayout.m

```diff
@@ -531,8 +531,8 @@ - (void)updateCellStateMaskWithAttribute:(MDCCollectionViewLayoutAttributes *)at
           (id<MDCCollectionViewEditingDelegate>)self.collectionView.dataSource;
 
       // Check if delegate can select during editing.
-      if ([editingDelegate respondsToSelector:@selector(collectionView:
-                                                  canSelectItemDuringEditingAtIndexPath:)]) {
+      if ([editingDelegate respondsToSelector:@selector
+                           (collectionView:canSelectItemDuringEditingAtIndexPath:)]) {
```

Comment:
This looks like the automatic formatter, do we normally break-up the @selector from the method name?